foreman_leapp.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
 
   s.add_dependency 'foreman_remote_execution', '>= 3.2', '< 5.0'
-  s.add_dependency 'foreman_ansible', '~> 5.0'
+  s.add_dependency 'foreman_ansible', '~> 6.0'
```

Comment:
Shouldn't we switch to requiring a specific lower version and leaving the upper version unbounded? That's what we do in rex when depending on tasks and in ansible when depending on rex.

Reply:
You mean like:
`'foreman_ansible', '>= 6.0'`

Reply:
More like `'foreman_ansible', '>= 5.0'`, since the breaking change there was just a required foreman version bump